README.md

```diff
@@ -28,60 +29,43 @@ dependencies:
 
 ### Android
 
-   1. Open the Android project with Android Studio.
-   2. Make sure that your `MainActivity.kt` extends `FlutterFragmentActivity`.
-      ```kotlin
-      import io.flutter.embedding.android.FlutterFragmentActivity
+1.  Open the Android project with Android Studio.
+2.  Make sure that your `MainActivity.kt` extends `FlutterFragmentActivity`.
 
-      class MainActivity: FlutterFragmentActivity() {
-        // Your code...
-      }
-      ```
-   3. Make sure the min sdk is 21 in your `app` `build.gradle`:
-   ```groovy
-     defaultConfig {
-        minSdkVersion 21
+    ```kotlin
+    import io.flutter.embedding.android.FlutterFragmentActivity
+
+    class MainActivity: FlutterFragmentActivity() {
+      // Your code...
     }
+    ```
+
+3.  Make sure the min sdk is 21 in your `app` `build.gradle`:
+
+```groovy
+  defaultConfig {
+     minSdkVersion 21
+ }
+```
+
+4. In your `Project` `build.gradle` include the Jitpack repository:
+   ```groovy
+   maven {
+     url 'https://www.jitpack.io'
+   }
```

Comment:
@adam-abbas can you test if we can just add it in the build.gradle of the plugin here: https://github.com/Mopinion-com/mopinion_flutter_integration_plugin/blob/develop/android/build.gradle#L6-L9

Reply:
Does not work unfortunately:
<img width="940" alt="image" src="https://github.com/Mopinion-com/mopinion_flutter_integration_plugin/assets/79903878/378f87ed-9353-4884-b02c-73b72032d6f1">